### PR TITLE
[Android] Bank Accounts support

### PR DESCRIPTION
**Pull request by paynd**
Fix for #28, #61 
Update `example` app design
Upgrade `react-native` to `0.44` in `example` app

### CODE REVIEW

File: example/src/scenes/CustomBankScreen.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+          onPress={this.handleBankAccountPayPress}
+          {...testID('customAccountButton')}
+        />
+        <View
+          style={styles.token}
+          {...testID('customAccountToken')}>
+          {token &&
+          <Text style={styles.instruction}>
+            Token: {token.tokenId}
+          </Text>
```
Comment (itsmepetrov) [inline example/src/scenes/CustomBankScreen.js]: Fix indent

Comment (paynd) [inline example/src/scenes/CustomBankScreen.js]: fixed

File: README.md [hunk 2]
```diff
@@ -436,6 +461,49 @@ const token = await stripe.createTokenWithCard(params)
...
+
+* `routingNumber` _String_ (Required) - The routing number of this account.
+* `accountNumber` _String_ (Required) - The account number for this BankAccount.
+* `countryCode`  _String_ (Required) - The two-letter country code that this account was created in.
+* `currency`  _String_ (Required) - The currency of this account.
+* `accountHolderName` _String_ - The account holder's name.
+* `accountHolderType` _String_ - the bank account type. Can be one of: `company`|`individual`.
+* `fingerprint` _String_ - The account fingerprint.
+* `bankName` _String_ - The name of bank.
+* `last4` _String_ - The last four digits of the account number.
```
Comment (itsmepetrov) [inline README.md]: `params` does not contain `fingerprint`, `bankName` and `last4`; `routingNumber` is not required and should be placed after required fields.

File: README.md [hunk 3]
```diff
@@ -157,6 +157,20 @@ An object with the following keys:
 * `addressCountry` _String_ - The cardholder’s country.
 * `addressZip` _String_ - The cardholder’s zip code.
 
+##### `bankAccount`
```
Comment (itsmepetrov) [inline README.md]: add `bankAccount` field to the `token` fields above

File: example/src/scenes/CustomBankScreen.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+      routingNumber: '110000000', // required field, 9 digits
+      accountNumber: '000123456789', // required field
+      countryCode: 'us', // required field
+      currency: 'usd', // required field
+      accountType: 'individual', // "company" or "individual";
+      accountHolderName: 'Test holder name',
+      accountHolderType: 'company',
+      fingerprint: 'dghajaryktskat',
+      bankName: 'Sberbank',
+      last4: '0000',
```
Comment (itsmepetrov) [inline example/src/scenes/CustomBankScreen.js]: remove `accountType`, `fingerprint`, `bankName` and `last4` fields, also remove all `required field` comments

File: README.md [hunk 5]
```diff
@@ -436,6 +462,45 @@ const token = await stripe.createTokenWithCard(params)
...
+##### `params`
+
+An object with the following keys:
+
+* `routingNumber` _String_ (Required) - The routing number of this account.
+* `accountNumber` _String_ (Required) - The account number for this BankAccount.
+* `countryCode`  _String_ (Required) - The two-letter country code that this account was created in.
+* `currency`  _String_ (Required) - The currency of this account.
+* `accountHolderName` _String_ - The account holder's name.
+* `accountHolderType` _String_ - the bank account type. Can be one of: `company`|`individual`.
```
Comment (itsmepetrov) [inline README.md]: `routingNumber` is not required and should be placed after required fields.

File: README.md (hunk 3 above)
Comment (paynd) [inline README.md]: Fixed.

File: example/src/scenes/CustomBankScreen.js (hunk 4 above)
Comment (paynd) [inline example/src/scenes/CustomBankScreen.js]: Fixed.

File: README.md (hunk 5 above)
Comment (paynd) [inline README.md]: Fixed.

**Comment by itsmepetrov** [comment]
Looks good to merge

**Pull request merged**